cms-adapter/src/Rsbc.Dmf.CaseManagement.Tests/Startup.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Rsbc.Dmf.CaseManagement.Dynamics;
+
+namespace Rsbc.Dmf.CaseManagement.Tests
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var configuration = new ConfigurationBuilder()
+                .AddUserSecrets<Service.Startup>() // Add secrets from the service.
+                .AddEnvironmentVariables()
+                .Build();
+
+            services.AddDistributedMemoryCache();
```

Comment:
What is the reason for the distributed memory cache here?

Reply:
without it, I get this error:

_System.InvalidOperationException : Unable to resolve service for type 'Microsoft.Extensions.Caching.Distributed.IDistributedCache' while attempting to activate 'Rsbc.Dmf.CaseManagement.Dynamics.CachedADFSSecurityTokenProvider'._